tasmoadmin/config.json

```diff
@@ -16,20 +16,23 @@
   "boot": "auto",
   "hassio_api": true,
   "hassio_role": "default",
+  "ingress": true,
+  "ingress_port": 8099,
   "map": [
     "ssl"
   ],
   "ports": {
-    "9541/tcp": 9541
+    "9541/tcp": null
```

Comment:
That seems like a bad idea. How do the firmware updates get across?

Reply:
TBH, I haven't used the firmware update process. Initially I figured it would go through the ingress, but I guess that wouldn't work since it would need to be authenticated.